id/guide/installation.md

```diff
@@ -16,59 +16,84 @@ description: Memulai Nuxt.js itu ternyata sangat mudah. Satu proyek sederhana ha
-
-lalu instal the dependencies:
+atau dengan [yarn](https://yarnpkg.com/en/):
 
 ```bash
-$ cd <nama-proyek>
-$ npm install
+yarn create nuxt-app <my-project>
 ```
 
-dan jalankan proyek dengan:
+Kemudian akan muncul beberapa pertanyaan:
+
+1. Pilih framework untuk integrasi server-side:
+
+- None (Nuxt default server)
+- [Express](https://github.com/expressjs/express)
+- [Koa](https://github.com/koajs/koa)
+- [Hapi](https://github.com/hapijs/hapi)
+- [Feathers](https://github.com/feathersjs/feathers)
+- [Micro](https://github.com/zeit/micro)
+- [Adonis](https://github.com/adonisjs/adonis-framework) (WIP)
+
+2. Pilih framework UI kesukaan anda:
+
+- None (feel free to add one later)
+- [Bootstrap](https://github.com/bootstrap-vue/bootstrap-vue)
+- [Vuetify](https://github.com/vuetifyjs/vuetify)
+- [Bulma](https://github.com/jgthms/bulma)
+- [Tailwind](https://github.com/tailwindcss/tailwindcss)
+- [Element UI](https://github.com/ElemeFE/element)
+- [Ant Design Vue](https://github.com/vueComponent/ant-design-vue)
+- [Buefy](https://buefy.github.io)
+
+3. [Mode Nuxt yang anda inginkan (`Universal` or `SPA`)](https://nuxtjs.org/guide/release-notes#better-spa-experience)
+4. Menambah [axios module](https://github.com/nuxt-community/axios-module) untuk mempermudah melakukan HTTP request pada aplikasi anda.
+5. Menambah [EsLint](https://eslint.org/) untuk memformat coding anda dengan lint pada saat menyimpan.
+6. Menambah [Prettier](https://prettier.io/) untuk memformat coding anda dengan prettier pada saat menyimpan.
+
+Setelah menjawab semua pertanyaan semua dependencies yang dibutuhkan akan terinstall dan langkah selanjut nya adalah menjalankan aplikasi dengan:
 
 ```bash
 $ npm run dev
 ```
 
-Kini aplikasi berjalan pada http://localhost:3000.
+Aplikasi sekarang berjalan pada http://localhost:3000.
 
 <div class="Alert">
 
-Nuxt.js akan memantau perubahan file di dalam direktori <code>pages</code>, jadi tidak perlu me-restart aplikasi saat menambahkan halaman baru.
+Nuxt.js akan memantau perubahan pada file anda yang berada dalam direktori <code>pages</code>, jadi anda tidak perlu menjalankan ulang aplikasi anda ketika menambahkan halaman baru.
 
 </div>
 
-Untuk mengetahui lebih banyak tentang struktur direktori sebuah proyek: [Dokumentasi Struktur Direktori](/guide/directory-structure).
+Untuk menjelajahi lebih lanjut mengenai struktur direktori pada projek : [Directory Structure Documentation](/guide/directory-structure).
 
-## Mulai dari awal
+## Memulai dari awal
 
-Membuat aplikasi Nuxt.js dari awal juga sangat mudah, hanya memerlukan *1 file dan 1 direktori*. Mari kita buat satu direktori kosong untuk mulai membuat aplikasinya:
+Membuat suatu aplikasi dengan Nuxt.js dari awal itu mudah, anda hanya membutuhkan _1 file dan 1 direktori_. Mari kita membuat direktor baru untuk memulai pengerjaan projek:
 
 ```bash
-$ mkdir <nama-proyek>
-$ cd <nama-proyek>
+$ mkdir <project-name>
```

Comment:
I think its better if it also translated into `<nama-projek>`

Reply:
thank you for your review, i've updated the translations